sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -1134,6 +1138,14 @@ export class MessageSession extends LinkEntity {
       const nextRenewalTimeout = calculateRenewAfterDuration(this.sessionLockedUntilUtc!);
       this._sessionLockRenewalTimer = setTimeout(async () => {
         try {
+          let receiverName;
+          if (this._receiver) {
+            receiverName = this._receiver.name;
+          } else {
+            throw new Error(
+              "AMQP receiver is closed or undefined. Cannot renew lock on non-existing receiver link"
```

Comment:
The discussion at https://github.com/Azure/azure-sdk-for-js/pull/2338#discussion_r276868989 applies here as well, and so I don't believe we need to throw the error here.

Reply:
done